manager/apis/app/v1alpha1/fybrikapplication_types.go

```diff
@@ -37,32 +34,34 @@ type DataRequirements struct {
 	// +required
 	Interface InterfaceDetails `json:"interface"`
 
-	// CopyRequrements include the requirements for copying the data
+	// FlowParams include the requirements for explicit requests to copy the data
```

Comment:
Is there some inconsistency here? The comment says: "FlowParams include the requirements for explicit requests to copy the data" 

- Does is relevant to `copy` only?
- The `FlowRequirements` comments say:

> FlowRequirements include the requirements specific to the flow (AR, any flow?)
> Note: Implicit copies done for data plane optimization by Fybrik do not use these parameters

Reply:
Will fix comment